frontend/packages/console-app/src/components/network-policies/network-policy-ports.tsx

```diff
@@ -22,90 +18,80 @@ export const NetworkPolicyPorts: React.FunctionComponent<NetworkPolicyPortsProps
-    some: <>{t('public~Certain ports')}</>,
-  };
-  const selectedKey = allPorts ? 'all' : 'some';
-
   return (
     <>
-      <div className="form-group co-create-networkpolicy__ports-type">
-        <label>
-          {direction === 'ingress' ? t('public~Allow traffic to') : t('public~Allow traffic from')}
-        </label>
-        <Dropdown
-          dropDownClassName="dropdown--full-width"
-          items={dropdownItems}
-          onChange={(key) => {
-            const isAllPorts = key === 'all';
-            setAllPorts(isAllPorts);
-            if (isAllPorts) {
-              onChange([]);
-            } else {
-              onChange([{ key: _.uniqueId('port-'), port: '', protocol: 'TCP' }]);
-            }
-          }}
-          selectedKey={selectedKey}
-          title={dropdownItems[selectedKey]}
-        />
-      </div>
-      {!allPorts && (
-        <div className="form-group co-create-networkpolicy__ports-list">
-          <label>{t('public~Ports')}</label>
-          {ports.map((port, idx) => {
-            const key = `port-${idx}`;
-            return (
-              <div className="pf-c-input-group" key={key}>
-                <Dropdown
-                  items={{
-                    TCP: <>TCP</>,
-                    UDP: <>UDP</>,
-                    SCTP: <>SCTP</>,
-                  }}
-                  title={port.protocol}
-                  name={`${key}-protocol`}
-                  className="btn-group"
-                  onChange={(protocol) => onSingleChange({ ...port, protocol }, idx)}
-                />
-                <input
-                  className="pf-c-form-control"
-                  onChange={(event) =>
-                    onSingleChange({ ...port, port: event.currentTarget.value }, idx)
-                  }
-                  placeholder="443"
-                  aria-describedby="ports-help"
-                  name={`${key}-port`}
-                  id={`${key}-port`}
-                  value={port.port}
-                />
-                <Button
-                  aria-label={t('public~Remove port')}
-                  className="co-create-networkpolicy__remove-port"
-                  onClick={() => onRemove(idx)}
-                  type="button"
-                  variant="plain"
-                >
-                  <MinusCircleIcon />
-                </Button>
-              </div>
-            );
-          })}
-          <div className="co-toolbar__group co-toolbar__group--left co-create-networkpolicy__add-port">
-            <Button
-              className="pf-m-link--align-left"
-              onClick={() => {
-                onChange([...ports, { key: _.uniqueId('port-'), port: '', protocol: 'TCP' }]);
-              }}
-              type="button"
-              variant="link"
-            >
-              <PlusCircleIcon className="co-icon-space-r" />
-              {t('public~Add port')}
-            </Button>
+      <FormFieldGroup
```

Comment:
Seen with Andrew, we should get rid of this section "accessible ports", it's redundant with the "port" title